common/JackNetInterface.h

```diff
@@ -241,14 +241,16 @@ namespace Jack
             {
                 InitAPI();
             }
-
+			#ifndef _WIN32
+			// FIXME: window static reference to fSlaveCounter not working while linking
```

Comment:
I'm a little but surprised with all this. Isn't the last jack2 release for windows compiled with visual c++?